tests/suites/test_suite_platform.function

```diff
@@ -32,91 +32,106 @@ void sleep_ms(int milliseconds)
     usleep(milliseconds * 1000);
 #endif
 }
-#endif
 
+#endif
 /* END_HEADER */
 
 /* BEGIN_DEPENDENCIES */
 
 /* END_DEPENDENCIES */
 
 /* BEGIN_CASE depends_on:MBEDTLS_HAVE_TIME */
-void time_get_milliseconds()
-{
-    mbedtls_ms_time_t current = mbedtls_ms_time();
-    (void) current;
-    /* This goto is added to avoid warnings from the generated code. */
-    goto exit;
-}
-/* END_CASE */
-
-/* BEGIN_CASE depends_on:MBEDTLS_HAVE_TIME */
-void time_get_seconds()
-{
-    mbedtls_time_t current = mbedtls_time(NULL);
-    (void) current;
-    /* This goto is added to avoid warnings from the generated code. */
-    goto exit;
-}
-/* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_HAVE_TIME */
+/* `time_delay_milliseconds()` and `time_delay_seconds()` might be fail due to
+ * `signal interrupt`, `discontinues jumps` and `incremental adjustment`. The
+ * test functions run 3 iteration to ignore the failures caused by those
+ * conditions.
+ * - `signal interrupt`  : That is introduced by `sleep_ms()`. `nanosleep()` and
+ *                         `usleep()` can be interrupted by signal.
+ * - `discontinues jumps`: `CLOCK_REALTIME` can be manually changed. When it
+ *                         happens during test, the test will be failed on
+ *                         Linux.
+ * - `incremental adjustment`: When `incremental adjustments` are performed
```

Comment:
I agree that it's good to handle incremental adjustments, since they can happen at any time during normal operation. I don't understand why apparently 3 iterations is a good number to handle incremental adjustments.

Reply:
When incremental adjustments starts in 1st iteration and end in 2nd iteration, both iterations might fail.